reference/forms/types/collection.rst

```diff
@@ -133,26 +133,29 @@ you need is this JavaScript code:
 .. code-block:: javascript
 
     // add-collection-widget.js
-    jQuery(document).ready(function () {
-        jQuery('.add-another-collection-widget').click(function (e) {
-            var list = jQuery(jQuery(this).attr('data-list-selector'));
-            // Try to find the counter of the list or use the length of the list
-            var counter = list.data('widget-counter') || list.children().length;
-
-            // grab the prototype template
-            var newWidget = list.attr('data-prototype');
-            // replace the "__name__" used in the id and name of the prototype
-            // with a number that's unique to your emails
-            // end name attribute looks like name="contact[emails][2]"
-            newWidget = newWidget.replace(/__name__/g, counter);
-            // Increase the counter
-            counter++;
-            // And store it, the length cannot be used if deleting widgets is allowed
-            list.data('widget-counter', counter);
-
-            // create a new list element and add it to the list
-            var newElem = jQuery(list.attr('data-widget-tags')).html(newWidget);
-            newElem.appendTo(list);
+    document.addEventListener('DOMContentLoaded', function() {
+        document.querySelectorAll('.add-another-collection-widget').forEach(function (elem) {
+            elem.addEventListener('click', function(e) {
+                let list = document.querySelector(elem.dataset.listSelector);
```

Comment:
`let list'. could be `const list`? And below, `let newElem` could be `const newElem`?